src/openforms/fixtures/default_admin_index.json

```diff
@@ -334,5 +334,16 @@
         "name": "Audit log",
         "link": "/admin/logging/timelinelogproxy/auditlog/"
     }
+},
+{
+    "model": "admin_index.applink",
+    "fields": {
+        "order": 5,
+        "app_group": [
+            "handige-links"
```

Comment:
better to move this to the configuration tab - this app group is meant for external URLs for meta-stuff, not for internal Open Forms instance things.